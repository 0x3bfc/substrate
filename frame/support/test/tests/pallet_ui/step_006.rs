#[frame_support::pallet(Example)]
mod pallet {
	#[pallet::trait_]
	pub trait Foo {}
}

fn main() {
}
