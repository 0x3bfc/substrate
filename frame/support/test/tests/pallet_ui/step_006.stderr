error: expected `Config`
 --> $DIR/step_006.rs:4:12
  |
4 |     pub trait Foo {}
  |               ^^^
